### PR TITLE
Menu(feat):Adding menu option in sprint details on the board

### PR DESCRIPTION
**Pull request by anu-aj**
### Functionality:
Add menu option , that helps to navigate to 
- Active Sprint
- Backlog
- Deleted Task

### Solution:
A dropdown menu added to the board that consists of sprint details

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:

1. ng serve
2. firebase emulators:start

### Screenshots 

Screenshot (641)

Screenshot (642)

### CODE REVIEW

File: src/app/body/board/board.component.ts [hunk 1]
```diff
@@ -31,15 +31,15 @@ export class BoardComponent implements OnInit {
...
-        this.readSprintData();
+    this.applicationSettingsService.getTeamDetails().subscribe(teams => {
+      this.teamData = teams;
+      teams.forEach(element => {
+        if (element.TeamId == this.selectedTeamId) {
+          this.teamCurrentSprintNumber = element.CurrentSprintId;
+        }
       });
+      this.readSprintData();
+    });
```
Comment (viv9k) [inline src/app/body/board/board.component.ts:42]: revert these changes, there is no change in this code.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please complete the required changes

**Review by viv9k — APPROVED** [review]
ship it

**Comment by viv9k** [comment]
Change target branch to dev-angular

**Comment by anu-aj** [comment]
> Change target branch to dev-angular

Okay

**Review by viv9k — APPROVED** [review]
Ship it.
